src/signframework/framework_utils.h

```diff
@@ -63,6 +63,7 @@ typedef struct tdProjectConfig {
     char        **senderemails; /* array of associated sender email addresses */
     size_t		notificationListCount;	/* number of notification receivers */
     char		**notificationList;	/* array of notification names */
+    Arguments   additionalArgs;     /* Additional sign tool arguments specified in project config */
```

Comment:
ProjectConfig_Init does a full memset of the structure which will cover the init of Arguments.  However to avoid any future issues I will add a call to Argument_Init inside there as well.